libtiny_tui/src/msg_area/line.rs

```diff
@@ -149,8 +149,12 @@ impl Line {
         self.len_chars += 1;
     }
 
-    /// How many lines does this take when rendered? O(n) where n = number of
-    /// split positions in the line (i.e. whitespaces).
```

Comment:
Why was this comment removed? Is this not valid anymore?

Reply:
Yeah, it's O(n) where n is the size of the buffer